scripts/globals/items/rabbit_cap.lua

```diff
@@ -0,0 +1,28 @@
+-----------------------------------
+-- ID: 26788
+-- Item: Rabbit Cap
+-- When used, you will obtain 1-2 random initial eggs
+-----------------------------------
+require("scripts/globals/msg")
+require("scripts/globals/items")
+-----------------------------------
+local itemObject = {}
+
+itemObject.onItemCheck = function(target)
+    local result = 0
+    if target:getFreeSlotsCount() == 0 then
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+
+    return result
+end
+
+itemObject.onItemUse = function(target)
+    target:addItem(math.random(xi.items.A_EGG, xi.items.Z_EGG))
+
+    if math.random(5) > 4 then
```

Comment:
Minor nitpick of mine, but I prefer defining a (min, max) range when using randoms.
I don't really know at glance or memory if `math.random(5)` returns 0-5 or 1-5

Reply:
> Minor nitpick of mine, but I prefer defining a (min, max) range when using randoms. I don't really know at glance or memory if `math.random(5)` returns 0-5 or 1-5

starts at 1 when only 1 parameter - can't start at 0 unless explicitly told to, or if doing the 0 to 1 float type.

Reply:
its still worth using both params just for clarity